### ISSUE TITLE
Explore adding further guidance on a WCAG criterion to the website

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#4681]
## Brief
Explore adding further guidance to the Design System website on WCAG 2.2 criteria that were not covered by the new 2.2 criteria. This first brief will focus on one criterion and draft guidance on it, with a stretch goal to publish.

## Epic lead
Anika

## Squad
Ciandelle, Kelly

**Comment by selfthinker** [comment]
Closing as we've done all the non-stretch tasks.
The outcomes of this work are in #4684.

In short, we found it would be worth doing. And we wouldn’t need to do it for all 56 success criteria, just doing it for the top 5 failing ones would cover 49% of fails.
